monkey/infection_monkey/network/elasticfinger.py

```diff
@@ -41,8 +40,7 @@ def get_host_fingerprint(self, host):
                 host.services[ES_SERVICE]['cluster_name'] = data['cluster_name']
                 host.services[ES_SERVICE]['name'] = data['name']
                 host.services[ES_SERVICE]['version'] = data['version']['number']
-                VictimHostTelem('T1210', ScanStatus.SCANNED.value,
-                                host, {'port': ES_PORT, 'service': 'Elastic'}).send()
+                host.services[ES_SERVICE].update(self.format_service_info(url=url))
```

Comment:
This will be irrelevant because of other comments, but the right thing would be to use a port instead of the url since it doesn't have any interesting info.

Reply:
It does. The url looks like 1.1.1.1:9200/_search?pretty (has a weird extension). And it's more convenient to copy-paste the url to check the endpoint instead of building it yourself.